flexget/utils/imdb.py

```diff
@@ -22,6 +23,7 @@
 
 # this makes most of the titles to be returned in english translation, but not all of them
 requests.headers.update({'Accept-Language': 'en-US,en;q=0.8'})
+requests.headers.update({'X-Forwarded-For': '24.110.%d.%d' % (random.randint(0, 254), random.randint(0,254))})
```

Comment:
Current code already has harcoded header trying to enforce en-US (probably ineffective), and we would default to english anyway...

I can work on making it configurable, but if you don't mind, I'd prefer to work on a few other things before getting back to this. :)